### PR TITLE
Add track moviment

### PR DESCRIPTION
**Pull request by danielpassos**
See: [AGDROID-599](https://issues.jboss.org/browse/AGDROID-599)

### CODE REVIEW

File: AeroDoc/app/src/main/java/org/jboss/aerogear/android/cookbook/aerodoc/AeroDocApplication.java [hunk 1]
```diff
@@ -148,7 +148,7 @@ public void registerDeviceOnPushServer(String alias) {
             registrar.register(getApplicationContext(), new Callback<Void>() {
                 @Override
                 public void onSuccess(Void data) {
-                    Log.d("GCM", "Registered");
+                    Log.d(TAG, "Device registered on UPS");
```
Comment (jcesarmobile) [inline AeroDoc/app/src/main/java/org/jboss/aerogear/android/cookbook/aerodoc/AeroDocApplication.java]: There is no TAG on this class

File: AeroDoc/app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -22,6 +22,9 @@
     <string name="accept">Accept</string>
     <string name="dismiss">Dismiss</string>
     <string name="available">Available</string>
+    <string name="location_denied">Without permission there is no track moviment</string>
```
Comment (jcesarmobile) [inline AeroDoc/app/src/main/res/values/strings.xml]: Change the text to something like "Without permission the app can't track movement"

**Comment by danielpassos** [comment]
@jcesarmobile Fixed!

**Comment by matzew** [comment]
:+1:

**Comment by jcesarmobile** [comment]
I've been able to run the AeroDoc server now and it works. 
The changes are ok, so LGTM

**Pull request merged**